tomviz/LoadDataReaction.cxx

```diff
@@ -275,6 +282,23 @@ void LoadDataReaction::dataSourceAdded(DataSource* dataSource)
     view = ActiveObjects::instance().activeView();
   }
 
+  ActiveObjects::instance().setMoveObjectsMode(oldMoveObjectsEnabled);
+
+  if (defaultModules) {
+    addDefaultModules(dataSource);
+  } else {
+    auto pqview = tomviz::convert<pqView*>(view);
+    pqview->resetDisplay();
```

Comment:
I see now, but still don't understand why we would reset the display/render when nothing that can be rendered is being added. I don't see anything but a data source being added, and so reset display/render should not be needed, and would have to be called later when we add a module.